src/libmongoc/src/mongoc/mongoc-topology.c

```diff
@@ -418,6 +430,15 @@ mongoc_topology_new (const mongoc_uri_t *uri, bool single_threaded)
    if (mongoc_uri_get_option_as_bool (
           topology->uri, MONGOC_URI_LOADBALANCED, false)) {
       init_type = MONGOC_TOPOLOGY_LOAD_BALANCED;
+      if (topology->single_threaded) {
+         /* Cooldown only applies to server monitoring for single-threaded
+          * clients. In load balanced mode, the topology scanner is used to
+          * create connections. The cooldown period does not apply. A network
+          * error to a load balanced connection does not imply subsequent
+          * connection attempts will be to the same server and that a delay
+          * should occur. */
```

Comment:
The [cooldown period](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-monitoring.rst#cooldownms) is described in the server monitoring specification.

Single threaded connections are created in the topology scanner so go through the machinery that does SDAM monitoring. But in load balanced mode, once a connection is created, it is never monitored. If the connection is closed (e.g. through a network error) the scanner should create a new connection as soon as one is requested.